content/pages/newsletter/index.md

```diff
@@ -0,0 +1,19 @@
+---
+alias_paths:
```

Comment:
Going to merge this just wanted to point out that alias_paths is the list of URLs that should be created as redirects to this page. This is to facilitate a redirect from old readme.io URLs to new paths.

So in this case this should have said:

```
alias_paths:
  - /docs/developer-newsletter
```